server/src/main/java/org/elasticsearch/search/aggregations/metrics/GeoCentroid.java

```diff
@@ -8,14 +8,7 @@
 
 package org.elasticsearch.search.aggregations.metrics;
 
-import org.elasticsearch.common.geo.GeoPoint;
-import org.elasticsearch.search.aggregations.Aggregation;
-
 /**
  * Interface for {@link GeoCentroidAggregator}
  */
-public interface GeoCentroid extends Aggregation {
-    GeoPoint centroid();
-
-    long count();
-}
+public interface GeoCentroid extends CentroidAggregation {}
```

Comment:
This was because I wrote the CartesianCentroid stuff, and then Ignacio wanted the pure refactoring parts pulled out. The Cartesian work is in a separate PR on top of this work. If I move this change into that PR it will cause that PR to contain a lot of test code changes that are geo-specific (not cartesian). My goal is for this PR to contain only refactoring (moving Geo-specific code around) and the other PR to contain primarily Cartesian specific stuff (minimize Geo changes). So for the purpose of that split, this is a good change :-)